### PR TITLE
bootstrap deployment file

### PR DESCRIPTION
**Pull request by evaldofelipe**
Signed-off-by: evaldofelipe <contato@evaldofelipe.com>

For local tests, just execute the file: ```/bin/bash ./ci/solidity-docs.sh && yarn start```
To edit the contracts docs, edit the template at: ```ci/contract.hbs```

### CODE REVIEW

**Comment by mrice32** [comment]
> Signed-off-by: uma-mining [contato@evaldofelipe.com](mailto:contato@evaldofelipe.com)

@evaldofelipe just out of curiosity, why are you commits signed off as uma-mining rather than Evaldo Felipe?

File: config/navbar.js [hunk 1]
```diff
@@ -47,8 +47,12 @@ module.exports = {
...
-    { to: "community/community-overview", 
-      label: "Community", 
+    { to: "community/community-overview",
+      label: "Community",
+      position: "left",
+      activeBaseRegex: "(users|community)" },
+    { to: "contracts/Migrations",
+      label: "Contracts",
       position: "left",
       activeBaseRegex: "(users|community)" },
```
Comment (smb2796) [inline config/navbar.js:57]:
```suggestion
      activeBaseRegex: "(contracts)" },
```

**Comment by evaldofelipe** [comment]
> > Signed-off-by: uma-mining [contato@evaldofelipe.com](mailto:contato@evaldofelipe.com)
> 
> @evaldofelipe just out of curiosity, why are you commits signed off as uma-mining rather than Evaldo Felipe?

Sorry @mrice32, my local configs here was different. Fixed, thanks!

File: ci/contract.hbs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+```
+{{name}}({{args}}){{#if outputs}} → {{outputs}}{{/if}} ({{visibility}})
+```
+{{/if}}
+{{{natspec.userdoc}}}
+
+{{{natspec.devdoc}}}
+
+{{#if natspec.params}}
+Parameters:
```
Comment (chrismaree) [inline ci/contract.hbs:21]: I think this should be a 4th level heading. Consider making all the `Parameters` equal to `#### Parameters:`

**Review by chrismaree — APPROVED** [review]
This is a pretty awesome start! really cool to see this being built out. I think there is a fair amount of future improvements we can make to this but it is a really good start!

**Review by smb2796 — APPROVED** [review]

File: ci/contract.hbs (hunk 2 above)
Comment (mrice32) [reply ci/contract.hbs:21]: +1

**Review by mrice32 — APPROVED** [review]
LGTM! Awesome to see this working. We will probably want to iterate on the template over time.

**Pull request merged**